src/Api/Controllers/OrganizationExportController.cs

```diff
@@ -30,19 +30,46 @@ public class OrganizationExportController : Controller
     }
 
     [HttpGet("export")]
-    public async Task<OrganizationExportResponseModel> Export(Guid organizationId)
+    public async Task<IActionResult> Export(Guid organizationId)
     {
         var userId = _userService.GetProperUserId(User).Value;
 
         IEnumerable<Collection> orgCollections = await _collectionService.GetOrganizationCollections(organizationId);
         (IEnumerable<CipherOrganizationDetails> orgCiphers, Dictionary<Guid, IGrouping<Guid, CollectionCipher>> collectionCiphersGroupDict) = await _cipherService.GetOrganizationCiphers(userId, organizationId);
 
-        var result = new OrganizationExportResponseModel
+        var clientVersion = this.HttpContext.Request.Headers["Bitwarden-Client-Version"];
```

Comment:
While this technically works, we should extract the version header logic to `CurrentContext`.

Reply:
Good thinking, I've done that!